docs/deploy/agent-engine/asp.md

```diff
@@ -0,0 +1,222 @@
+# Deploy to Agent Engine with Agent Starter Pack
+
+<div class="language-support-tag" title="Vertex AI Agent Engine currently supports only Python.">
+    <span class="lst-supported">Supported in ADK</span><span class="lst-python">Python</span>
+</div>
+
+This deployment procedure describes how to perform a deployment using the
+[Agent Starter Pack](https://github.com/GoogleCloudPlatform/agent-starter-pack)
+(ASP) and the ADK command line interface (CLI) tool. Using ASP for deployment
+is an accelerated path you should use it for _*development and testing*_ only.
```

Comment:
Awkward wording in "is an accelerated path you should use it for development and testing".